### ISSUE TITLE
Node.js Foundation Technical Steering Committee (TSC) Meeting 2016-07-14

### ISSUE THREAD

**Issue by rvagg** [nodejs/TSC#118]
## Time

**UTC Thu 14-Jul-2016 20:00**:

| Timezone      | Date/Time             |
|---------------|-----------------------|
| US / Pacific  | Thu 14-Jul-2016 13:00 |
| US / Mountain | Thu 14-Jul-2016 14:00 |
| US / Central  | Thu 14-Jul-2016 15:00 |
| US / Eastern  | Thu 14-Jul-2016 16:00 |
| Amsterdam     | Thu 14-Jul-2016 22:00 |
| Berlin        | Thu 14-Jul-2016 22:00 |
| Moscow        | Thu 14-Jul-2016 23:00 |
| Tokyo         | Fri 15-Jul-2016 05:00 |
| Sydney        | Fri 15-Jul-2016 06:00 |

Or in your local time:
* http://www.timeanddate.com/worldclock/fixedtime.html?msg=Node.js+Foundation+Technical%20Steering%20Committee%20(TSC)+Meeting+2016-07-14&iso=20160714T20
* or http://www.wolframalpha.com/input/?i=8pm+UTC%2C+Jul+14%2C+2016+in+local+time

## Links

* **Minutes Google Doc**: <https://docs.google.com/document/d/1M29qRvgLU5LedQZ9kpZfWzzlDbNvWB9RolWBcD5ZBco>
GROUP_NAME="Technical Steering Committee (TSC)"
* _Previous Minutes Google Doc: <https://docs.google.com/document/d/1eUpQNVdupAlsyvVVEnu8za6d8aBhOv9bEZK9FVWm8ds>_

## Agenda

Extracted from **tsc-agenda** labelled issues and pull requests from the **nodejs org** prior to the meeting.

### nodejs/TSC

* Define "Node.js core" - Mark II [#113](https://github.com/nodejs/TSC/issues/113)
* Regular roll-up reporting for active working groups [#109](https://github.com/nodejs/TSC/issues/109)
* Define "Node.js core" [#84](https://github.com/nodejs/TSC/issues/84)

## Invited

* Ben Noordhuis @bnoordhuis (TSC)
* Bryan Hughes @nebrius (TSC)
* Chris Dickinson @chrisdickinson (TSC)
* Colin Ihrig @cjihrig (TSC)
* Jeremiah Senkpiel @Fishrock123 (TSC)
* Fedor Indutny @indutny (TSC)
* James M Snell @jasnell (TSC)
* Josh Gavant @joshgav (observer/Microsoft)
* Julien Gilli @misterdjules (TSC)
* Mikeal Rogers @mikeal (observer/Node.js Foundation)
* Brian White @mscdex (TSC)
* Alexis Campailla @orangemocha (TSC)
* Bert Belder @piscisaureus (TSC)
* Rod Vagg @rvagg (TSC)
* Shigeki Ohtsu @shigeki (TSC)
* Trevor Norris @trevnorris (TSC)
* William Kapke @williamkapke (observer)

## Notes

The agenda comes from issues labelled with `tsc-agenda` across **all of the repositories in the nodejs org**. Please label any additional issues that should be on the agenda before the meeting starts.

## Joining the meeting

Uberconference; participants should have the link & numbers, contact me if you don't.

## Public participation

We stream our conference call straight to YouTube so anyone can listen to it live, it should start playing at **https://www.youtube.com/c/nodejs+foundation/live** when we turn it on. There's usually a short cat-herding time at the start of the meeting and then occasionally we have some quick private business to attend to before we can start recording & streaming. So be patient and it should show up.

Many of us will be on IRC in #node-dev on Freenode if you'd like to interact, we have a Q/A session scheduled at the end of the meeting if you'd like us to discuss anything in particular. @nodejs/collaborators in particular if there's anything you need from the TSC that's not worth putting on as a separate agenda item, this is a good place for it.

**Comment by rvagg** [comment]
The TSC would like to invite @joshgav and @williamkapke as observers to these meetings given the great value they've been adding at this administrative end of our activities.

Additionally, we haven't done this formally (haven't even actually discussed this tbh) but I'd like to invite @nodejs/ctc members who are _not_ members of the TSC given the timing of the split between the two groups to join these meetings if they are interested. We could always do with more hands on this work and we still need to figure out the exact relationship with the CTC. So that would be @ChALkeR, @evanlucas, @mhdawson (who has already been joining us and been a great help), @ofrobots, @Trott. Once we figure it all out (you can help!) I'd expect there to be a logical path for any of you to join the TSC if your interests are in this area and we refine the member list (shedding those who currently don't want to be involved).

**Comment by bnoordhuis** [comment]
I probably won't be joining tonight.  I have to get up awfully early tomorrow morning.

**Comment by williamkapke** [comment]
Great Thanks.

Please reflect on this (old) open issue about @zcat being an observer also: #83. If it is no longer relevant, maybe someone could update/close the issue.
